src/Runner/TestSuiteSorter.php

```diff
@@ -36,6 +36,11 @@
      */
     public const ORDER_DEFECTS_FIRST = 3;
 
+    /**
+     * @var int
+     */
+    public const ORDER_TIME = 4;
```

Comment:
Maybe rename this to `ORDER_DURATION`?

Reply:
`ORDER_EXECUTION_TIME` maybe?

Reply:
Yes, `DURATION` would be nicer as the word time also pops up in e.g. `START_TIME` and `END_TIME`. Just a small thing :)

Reply:
`ORDER_DURATION` it is then :-) Thanks!